indra/newview/llvoavatarself.h

```diff
@@ -55,11 +55,11 @@ class LLVOAvatarSelf :
 public:
 	LLVOAvatarSelf(const LLUUID &id, const LLPCode pcode, LLViewerRegion *regionp);
 	virtual 				~LLVOAvatarSelf();
-	virtual void			markDead();
-	virtual void 			initInstance(); // Called after construction to initialize the class.
+	void			        markDead() override;
+	void 			        initInstance() override; // Called after construction to initialize the class.
 	void					cleanup();
 protected:
-	/*virtual*/ BOOL		loadAvatar();
+	BOOL		            loadAvatar() override;
```

Comment:
I approve the use of the the `override` keyword!

Reply:
The catch is that once you use it once, you have to make sure you use it everywhere. (Bit me on the mac build)